index.src.html

```diff
@@ -382,8 +397,8 @@ <h4 id="examples-federated-signin">Federated Sign-in</h4>
 
     <h4 id="examples-post-signin">Post-sign-in Confirmation</h4>
 
-    To ensure that credentials are persisted correctly, it may be useful for a
-    website to tell the credential manager that sign-in succeeded.
+    To offer the user that credentials are persisted after a successful
```

Comment:
I am not in favor of that change. I see that the calling of store as a necessary but not sufficient condition to pop up a UI and think that the sentence is correct. "To ensure that credentials are persisted according to user expectations" is extremely abstract in my opinion.
@mikewest WDYT?